test/test_jit.py

```diff
@@ -15182,6 +15182,38 @@ def forward(self, x):
         with self.assertRaisesRegex(RuntimeError, 'has no attribute'):
             torch.jit.script(wrapped)
 
+    def test_rescripting_loaded_modules(self):
+        class InnerSubmod(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.register_buffer("foo", torch.ones(1))
+
+            def forward(self, x):
+                return x + x
+
+        class Inner(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.submod = InnerSubmod()
+
+            def forward(self, x):
+                return self.submod(x)
+
+        class Wrapper(nn.Module):
+            def __init__(self, inner):
+                super().__init__()
+                self.inner = inner
+
+            def forward(self, x):
+                # access inn eelem
+                return self.inner.submod(x) + self.inner.submod.foo
+
+        inner_module = torch.jit.script(Inner())
```

Comment:
```suggestion
        inner_module = torch.jit.script(Inner())
        wrapped = Wrapper(inner_module)
        self.checkModule(wrapped, torch.ones(1))
```
To be more faithful to the stated goal of this PR, I think we should run `checkModule` with this version of `inner_module`. We could also compare the outputs of `Wrapped(inner_module)` and `Wrapped(inner_module_loaded)`.

Reply:
good idea